src/Microsoft.AspNetCore.OData/Extensions/ODataRouteBuilderExtensions.cs

```diff
@@ -25,6 +25,382 @@ namespace Microsoft.AspNet.OData.Extensions
     /// </summary>
     public static class ODataRouteBuilderExtensions
     {
+        /// <summary>
+        /// Sets the <see cref="DefaultQuerySettings"/> in route builder.
+        /// </summary>
+        /// <param name="builder">The <see cref="IRouteBuilder"/>.</param>
+        /// <param name="defaultQuerySettings">The default query settings.</param>
+        public static IRouteBuilder SetDefaultQuerySettings(this IRouteBuilder builder, DefaultQuerySettings defaultQuerySettings)
```

Comment:
I am thinking Why do we need to set "Default" query setting? 

So, the public api maybe "SetQuerySettings" ?

Reply:
For back-compat, I'd like to keep it the same as AspNet, which uses "Default".